File: utility/msToQ15.java
public class msToQ15 {
    public static void main(String[] args) {
        int audioRate = 44100;
        double time_ms = 0;
        double result = 0;

        if(args.length > 0) {
            time_ms = Float.parseFloat(args[0]);
        } 
        
        result = (1 - Math.exp(- 1 / (audioRate * time_ms / 1000))) * 32767.0f;

        System.out.println("Q15 coefficient of " + time_ms + "ms is: " + result);
    }
}

File: utility/msToQ15.py
import sys
import math

def convert_coeff(time_ms : float):
    return round((1.0 - math.exp(-1.0 / (44100 * time_ms / 1000))) * 32767.0)

print(f"Q15 coefficient for {sys.argv[1]} ms is: ", convert_coeff(float(sys.argv[1])))